### PR TITLE
Add options for sensitivity where password is present. Fix docs for keychain kc_file

### PR DESCRIPTION
**Pull request by Brantone**
## Describe the Pull Request  
Minor modification on docs mentioning `kc_file` attribute rather than `keychain`
Add sensitivity options where passwords are applicable

## Justification  
Clear docs and password security

### CODE REVIEW

**Comment by jazaval** [comment]
Thanks, @Brantone! I've updated the target branch to our upcoming 5.0 release and fixed the merge. Also, do you have any objections to requiring the appropriate resource calls to be `sensitive`, or at least making `sensitive` true by default?

**Review by jweyer8 — APPROVED** [review]

File: resources/certificate.rb [hunk 1]
```diff
@@ -5,15 +5,16 @@
 
 property :certfile, String
 property :cert_password, String, sensitive: true
-property :keychain, String
+property :keychain, String, required: true
+property :kc_passwd, String, required: true, sensitive: true
```
Comment (Brantone) [inline resources/certificate.rb:9]: 👍 
Awesome, was thinking about doing similar on fork, but wasn't sure of use-case for `node['macos']['admin_password']`

**Comment by Brantone** [comment]
>  do you have any objections to requiring the appropriate resource calls to be sensitive, or at least making sensitive true by default?

That would makes sense to me. I was thinking of doing similar, but wasn't sure about maintaining expected behaviour and communicating to other impending change.

**Comment by jazaval** [comment]
> > do you have any objections to requiring the appropriate resource calls to be sensitive, or at least making sensitive true by default?
> 
> That would makes sense to me. I was thinking of doing similar, but wasn't sure about maintaining expected behaviour and communicating to other impending change.

We're bumping the major version of the cookbook, and assuming no one is consuming without a version pin for breaking changes (or is utilizing Policyfiles).

**Comment by Brantone** [comment]
I might have further doc tweaks coming in ... very minor.

File: resources/certificate.rb [hunk 2]
```diff
@@ -8,19 +8,22 @@
 property :keychain, String, required: true
 property :kc_passwd, String, required: true, sensitive: true
```
Comment (Brantone) [inline resources/certificate.rb:9]: I know already merged, just wondering about consistency in naming of "password" for `cert_password` vs. `cert_passwd` 🤔

**Pull request merged**